net.sourceforge.pmd.eclipse.plugin/src/main/java/net/sourceforge/pmd/eclipse/runtime/writer/IRuleSetWriter.java

```diff
@@ -44,5 +44,5 @@
      * @param ruleSet
      *            the ruleset to serialize
      */
-    void write(OutputStream outputStream, RuleSet ruleSet) throws WriterException;
+    void write(OutputStream outputStream, RuleSets ruleSet) throws WriterException;
```

Comment:
We should keep the write method, which just takes a single RuleSet.
And maybe we don't need to write multiple rulesets at all...

Reply:
Why wouldn't we want to write multiple rulesets out?